atlasdb-feign/src/test/java/com/palantir/atlasdb/http/ExceptionCountingRefreshingClientTest.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.http;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.util.function.Supplier;
+
+import org.junit.Test;
+
+import com.google.common.collect.ImmutableMap;
+
+import feign.Client;
+import feign.Request;
+import feign.Response;
+
+public class ExceptionCountingRefreshingClientTest {
+    private Supplier<Client> clientSupplier = (Supplier<Client>) mock(Supplier.class);
+    private Client client = mock(Client.class);
+
+    private Request request = Request.create("POST", "atlas.palantir.pt", ImmutableMap.of(), null, null);
+    private Response response = Response.create(204, "no content", ImmutableMap.of(), new byte[0]);
+    private Request.Options options = new Request.Options(1, 1);
```

Comment:
nit: These 3 look like they're constants, so they could probably be `static final`